### ISSUE TITLE
ListView separatorColor property no longer working in 2.5.0

### ISSUE THREAD

**Issue by chimon2000** [NativeScript/NativeScript#3574]
### Tell us about the problem
While attempting a test upgrade, I discovered that ListView's separator color properties no longer work as expected. 

**Expected Behavior**
When using `separatorColor="transparent",` the ListView's separators don't show.
When using `separator-color: transparent` in the CSS, the ListViews's separators don't show.

**Actual Behavior**
When attempting to set directly on the ListView, it does not work. 
When attempting to set to transparent in the CSS, an error is thrown.

### Which platform(s) does your issue occur on?
Android

### Please provide the following version numbers that your issue occurs with:
- CLI: 2.5.0
- Cross-platform modules: 2.5.0
- Platform: 2.5.0

### Please tell us how to recreate the issue in as much detail as possible.
You can recreate the issue with the Angular 2 starter `tns create TestUpgrade --ng`

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
Changing the template to the following
````
    <ListView [items]="items" class="" separatorColor="transparent">
        <template let-item="item">
            <Label [nsRouterLink]="['/item', item.id]" [text]="item.name"
                class="list-group-item"></Label>
        </template>
    </ListView>
````
the color remains the same. Changing the CSS

````
ListView {
 separator-color: transparent;
}
````

Causes an error to be thrown.

**Comment by mounirka** [comment]
I have the same problem, separatorColor is no more working.
Hope that this issue will be solve very soon, otherwise i will have downgrade my nativescript version

**Comment by bot2600** [comment]
same problem here

**Comment by tsonevn** [comment]
Hi @chimon2000,
Thank you for reporting this issue.
This behavior has been reproduced, when you try to set `separator-color` to  `transparent`. I was able to reproduce behavior only on Android.

This problem with `transparent` property on Android has been fixed with PR: [#3567](https://github.com/NativeScript/NativeScript/pull/3567). 
In the meantime, you could use `@next` modules, where this behavior has been fixed. You could also follow the below given steps.

1. `tns plugin remove tns-core-modules`

2. `tns plugin add tns-core-modules@next`

3. delete `platforms`, `node_modules` and `hooks` folder before rebuild

Regards,
@tsonevn

**Comment by abhayastudios** [comment]
Indeed `tns-core-modules@next` solves the issue :)

**Comment by chimon2000** [comment]
Awesome to hear that this issue was resolved so quickly! Will test

**Comment by SBD580** [comment]
Doesn't work with @next when using property and not css

**Comment by blakeyu** [comment]
Got the same issue, will be resolve in next release?

**Comment by tsonevn** [comment]
Hi @SBD580 @blakeyu ,

This issue has been fixed with `tns-core-modules` 2.5.1. Could you verify, whether you are using modules `2.5.1` in your project `package,json` file? 
If you still have this behavior provide some more info for your environment(`CLI`, `node`, `modules` version).
It will help if you could send us sample project, which could be debugged locally.

**Comment by blakeyu** [comment]
Thanks @tsonevn, it's working with tns-core-modules 2.5.1.

**Comment by josempbrites** [comment]
I now have this problem for iOS. I just updated to the latest version. Any suggestion ?

**Comment by tsonevn** [comment]
Hi @josempbrites, 
Could you try do delete `node_modules`, `platforms` and `hooks` folder from your project folder and to delete npm cache with `npm cache clear`? Then try to rebuild again the app with `tns run <platform_name>`.

**Comment by josempbrites** [comment]
Hi, @tsonevn, I did that but its not working. It only works if I set the property in the CSS file, like
`ListView  {
    separator-color: transparent;
}`

**Comment by bryanrq123** [comment]
separatorColor property does not work even using `tns-core-modules@next`

**Comment by tsonevn** [comment]
Hi all, 
I tested setting `separatorColor` property inline in the XML file on my side and I have to confirm that this is a real issue for both iOS and Android.

At that point, I  logged [a new issue](https://github.com/NativeScript/NativeScript/issues/3792) and you could keep track on it for further info.

**Comment by tedivm** [comment]
I can also confirm that this is still a bug for IOS. When I add this to the XML file it is completely ignored, but when I add it to CSS it works as expected (since the CSS work around exists this isn't a critical issue, although debugging it was difficult until I ran into this ticket).

**Comment by deepak4u2006** [comment]
Hi @tsonevn ,

I followed the steps

> Hi @chimon2000,
Thank you for reporting this issue.
This behavior has been reproduced, when you try to set separator-color to transparent. I was able to reproduce behavior only on Android.

This problem with transparent property on Android has been fixed with PR: #3567.
In the meantime, you could use @next modules, where this behavior has been fixed. You could also follow the below given steps.

tns plugin remove tns-core-modules

tns plugin add tns-core-modules@next

delete platforms, node_modules and hooks folder before rebuild

Regards,
@tsonevn
